frontend/pages/settings/dlna.vue

```diff
@@ -0,0 +1,249 @@
+<template>
+  <settings-page page-title="settings.dlna.dlna">
+    <template #actions>
+      <v-btn
+        rel="noreferrer noopener"
+        href="https://jellyfin.org/docs/general/networking/dlna.html"
+        target="_blank"
+      >
+        {{ $t('settings.help') }}
+      </v-btn>
+    </template>
+    <template #content>
+      <v-col cols="12" md="6" lg="5" class="py-4">
+        <v-card class="mb-4">
+          <v-list two-line flat>
+            <v-list-item-group>
+              <v-list-item
+                v-for="dlnaSettingName in Object.keys(bindToRelevantConfig)"
```

Comment:
```suggestion
                v-for="(key, value) in Object.entries(bindToRelevantConfig)"
```

Soy you don't need to do ``dlnaSettings[dlnaSettingName]`` down below

Reply:
As mentioned above this approach isn't workable.